Sources/SwiftDocC/Infrastructure/Workspace/FeatureFlags+Info.swift

```diff
@@ -0,0 +1,82 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+extension DocumentationBundle.Info {
+    /// A collection of feature flags that can be enabled from a bundle's Info.plist.
+    ///
+    /// This is a subset of flags from ``FeatureFlags`` that can influence how a documentation
+    /// bundle is written, and so can be considered a property of the documentation itself, rather
+    /// than as an experimental behavior that can be enabled for one-off builds.
+    ///
+    /// ```xml
+    /// <key>CDExperimentalFeatureFlags</key>
+    /// <dict>
+    ///     <key>ExperimentalOverloadedSymbolPresentation</key>
+    ///     <true/>
+    /// </dict>
+    /// ```
+    public struct BundleFeatureFlags: Codable, Equatable {
```

Comment:
Would it be possible to keep this type internal and solely rely on the decoded flags updating the `FeatureFlags`?

Reply:
If we want to add it to the initializer, or have the `featureFlags` field be publicly accessible like the other properties, then the data type needs to itself be publicly accessible. I think it's still worth having the flags accessible here, so it's obvious which flags were enabled via the Info.plist versus those that were enabled via CLI flags.

Reply:
What would be the use case for checking if a feature flag was enabled on the command line or in the Info.plist?

Reply:
I'm not sure offhand, but since this is public API i don't want to discount the possibility.